PlayCover/Utils/Shell.swift

```diff
@@ -23,30 +23,34 @@ class Shell: ObservableObject {
 		try task.run()
 
 		let data = try pipe.fileHandleForReading.readToEnd() ?? Data()
-		let output = String(data: data, encoding: .utf8)!
-
-		if print {
-			Log.shared.log(output)
-		}
-
-		task.waitUntilExit()
+        if let output = String(data: data, encoding: .utf8) {
+            if print {
+                Log.shared.log(output)
+            }
 
-		let status = task.terminationStatus
-		if status != 0 {
-            if pipeStdErr {
-                throw output
-            } else {
-                let errOutput: String
-                do {
-                    let errData = try errPipe.fileHandleForReading.readToEnd() ?? Data()
-                    errOutput = String(data: errData, encoding: .utf8)!
-                } catch {
-                    errOutput = "Command '\(command)' failed to execute."
+            task.waitUntilExit()
+
+            let status = task.terminationStatus
+            if status != 0 {
+                if pipeStdErr {
+                    throw output
+                } else {
+                    var errOutput = ""
+                    do {
+                        let errData = try errPipe.fileHandleForReading.readToEnd() ?? Data()
+                        if let errString = String(data: errData, encoding: .utf8) {
+                            errOutput = errString
+                        }
+                    } catch {
+                        errOutput = "Command '\(command)' failed to execute."
+                    }
+                    throw errOutput
                 }
-                throw errOutput
             }
-		}
-		return output
+            return output
+        } else {
+            return ""
```

Comment:
I think we should send a different message here, instead of returning an empty string.

Reply:
Its useful to add it but not @discardableResult may handle that